.cursor/prompts/코드스타일_가이드라인.md

```
@@ -0,0 +1,1213 @@
+```typescript
+// apiUtils.ts에서 자동 처리됨
+export function getAuthHeaders(): Record<string, string> {
+  const token = sessionStorage.getItem("accessToken");
+  return {
+    "Content-Type": "multipart/form-data",
+    ...(token ? { Authorization: `Bearer ${token}` } : {}),
+  };
+}
+```
+
+**RefreshToken 처리**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)